src/vim9expr.c

```diff
@@ -291,7 +291,7 @@ compile_class_object_index(cctx_T *cctx, char_u **arg, type_T *type)
     }
 
     class_T *cl = type->tt_class;
-    int is_super = type->tt_flags & TTFLAG_SUPER;
+    int is_super = ((type->tt_flags & TTFLAG_SUPER) == TTFLAG_SUPER);
```

Comment:
Don't like this notation?
```c
int is_super = !!(type->tt_flags & TTFLAG_SUPER);
```

It's already being used in a few places.
(I like it because I don't have to use keyword twice)

Reply:
This notation is already used in a few places in the code (e.g. IS_OBJECT_METHOD, IS_CONSTRUCTOR_METHOD, IS_ABSTRACT_METHOD).  You can also search for `& \(\k\+\)) == \1` and see that it is used in a few places.

Reply:
I understand that your way of writing is common. I'm not criticizing; I was just casually making a suggestion, so please don't worry about it. :ok_woman: